docs/3.1. Interoperability - NMOS IS-04.md

```diff
@@ -49,8 +49,8 @@ In order to populate the `subscription` attribute of IS-04 Senders and Receivers
 
 The Connection API supersedes the now deprecated method of updating the `/target` resource on Node API Receivers in order to establish connections. The two methods of operation are likely to co-exist until Version 2.0 of IS-04. As such the following best practice SHOULD be followed when both are in use:
 
-- Where a client updates the Node API subscription, the result on the Connection API SHOULD be the same as if the client had first staged the parameters and then called an immediate activation. That is to say that the new parameters will be reflected both in the staged and active endpoints of the Receiver.
-- Where a client updates a Connection API Receiver the active `sender_id` parameter SHOULD be populated in the Node API subscription parameter with key `sender_id`.
-- After a Connection API activation the corresponding Sender or Receiver's `version` property SHOULD be updated to the instant of the activation.
+- Where a client establishes a connection via the Node API `/target` resource, which updates the Receiver's `subscription` object, the result on the Connection API SHOULD be as if the client had staged the parameters and requested an immediate activation to enable the Receiver. That is to say that the new parameters, including the `sender_id` and with `master_enable` set to `true`, will be reflected in both the `/staged` and `/active` endpoints of the Receiver.
+- When the active parameters of a Sender or Receiver are modified via a Connection API activation, the Node API Sender or Receiver `subscription` object SHOULD be updated to reflect the active state. That is to say that the `active` attribute and the `receiver_id` or `sender_id` of the `subscription` object will match the `master_enable` and `receiver_id` or `sender_id` values in the Connection API `/active` endpoint.
```

Comment:
This previously didn't mention `receiver_id`.  Perhaps it should, but is this definitely only an editorial change?

Reply:
+1 to changing filename as suggested.

Reply:
The IS-05-02 ("IS-05 Interaction with IS-04") test suite checks everything mentioned since December 2018, so I think this is editorial.